parsl/tests/test_python_apps/test_join.py

```diff
@@ -97,7 +97,10 @@ def test_error():
     f = outer_error()
     e = f.exception()
     assert isinstance(e, JoinError)
+
+    assert len(e.dependent_exceptions_tids) == 1
```

Comment:
This makes me wonder if at least 2 and 3 dependent exception tids would also be correct.  Missing test or parameterization opportunity?

Reply:
that is the behaviour tested in test_two_errors right after

Reply:
Ah, indeed.  Already tested.  Should have expanded the file in GH.